Purchases/Purchasing/ReceiptFetcher.swift

```diff
@@ -80,7 +80,7 @@ private extension ReceiptFetcher {
             return nil
         }
 
-        Logger.debug(String(format: Strings.receipt.loaded_receipt, receiptURL as CVarArg))
```

Comment:
What was that `as CVarArg` for? 🤔

Reply:
I'm not sure, but my guess is that it was there as a way to be able to use `String(format:` with a `URL` type